tests/Feature/SearchPageTest.php

```diff
@@ -306,12 +310,123 @@ public function itShowsFamilyPrimaryCarersAlphabetically()
     }
 
     /** @test */
-    public function itPreventsAccessToLeftFamilyRegistrations()
+    public function itHasTheExpectedResultsPerPage()
     {
+        $centre = factory(App\Centre::class)->create();
+
+        // Create a User
+        $user =  factory(App\User::class)->create([
+            "name"  => "test user",
+            "email" => "testuser@example.com",
+            "password" => bcrypt('test_user_pass'),
+            "centre_id" => $centre->id,
+        ]);
+
+        // Create 25 random registrations, which should be well over the pagination limit.
+        $registrations = factory(App\Registration::class, 25)->create([
+            "centre_id" => $centre->id,
+        ]);
+
+        $this->actingAs($user)
+            ->visit(URL::route('service.registration.index'));
+
+
+        // Spot  and count the Registrations Family's primary carer names
+        $selector = 'td.pri_carer';
+        $this->assertCount(10, $this->crawler->filter($selector));
+    }
```

Comment:
Do you also want to see pagination links? Just in case they don;t render. We had that in Service MOMO for ages where the pagination link disappeared on page 2.

Reply:
Ignore me... I see that in next test!